index.html

```diff
@@ -129,7 +224,39 @@ <h2>
 <button class="see-pr" type="button">See project</button>
 
 </div>
-<div class="card">
+<div class="card" id="card-desktop">
```

Comment:
- [ ] Please I highly recommend sticking to the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34-848&t=Hq64bUbN7WFDPuHZ-0). 😃 

_**REQUIRED FIGMA DESIGN**_

![Screenshot (198)](https://user-images.githubusercontent.com/78104544/230510879-52d578a7-cdc6-43a6-bb71-a99bda5c8cc2.png)


_**YOUR DESIGN**_

- [ ] Please I highly recommend maintaining the gaps between the `daily-read`, `dev-lang` and `see-pr` classes in order to fit the Figma Design. Kindly implement these changes on all the cards in the `work` section. 😃 

![Screenshot (197)](https://user-images.githubusercontent.com/78104544/230510908-7e7d6d62-e433-4115-a01c-e9724143ad26.jpg)